src/scripts.js

```diff
@@ -90,18 +90,21 @@ const setTripsDestination = () => {
   let destinationObj = destinationRepo.findDestination(trip.destination);
   trip.destination = destinationObj;
   });
-}
+};
 
 const findYearlySpent = (travelersTrips) => {
   let yearlySpent = 0;
   travelersTrips.forEach(trip => {
```

Comment:
this could be a reduce with an accumulator

Reply:
new push has a reduce, works way better than before